src/main/java/sa/assertj/iterables/AreAtMost.java

```diff
@@ -0,0 +1,34 @@
+package sa.assertj.iterables;
+
+import org.assertj.core.api.Condition;
+import sa.assertj.Experiment;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AreAtMost extends Experiment {
+    static Experiment.DataProvider provider = (size, numOfSamples) -> {
+        Object[][] result = new Object[numOfSamples][];
+        for (int i = 0; i < numOfSamples; i++) {
+            ArrayList<Integer> list = new ArrayList<>();
+            Random rand = new Random();
+            list.add(1);
```

Comment:
I'm thinking about how to make it also not always be the same number here but maybe we can discuss that together.

Reply:
Maybe instead of using that element we can randomise until it's not 0? And then randomise the rest as we do now?